besu/src/test/java/org/hyperledger/besu/cli/BesuCommandTest.java

```diff
@@ -1587,6 +1587,20 @@ public void rpcMaxLogsRangeOptionMustBeUsed() {
     assertThat(commandErrorOutput.toString(UTF_8)).isEmpty();
   }
 
+  @Test
+  public void rpcGasCapOptionMustBeUsed() {
```

Comment:
Bit confused what this test name means...it's optional right? In what sense must it be used?

Reply:
This was added following the pattern that a lot of other CLI options follow. Agree that the name could be better but I take that it means that the option must be used when specified and throw no error.